bedrock/products/views.py

```diff
@@ -95,6 +95,28 @@ def vpn_download_page(request):
     return l10n_utils.render(request, template_name, context, ftl_files=ftl_files)
 
 
+@require_safe
+def vpn_mac_download_page(request):
+    template_name = "products/vpn/mac-download.html"
+    ftl_files = ["products/vpn/platform-post-download", "products/vpn/shared"]
```

Comment:
Would adding this `reference` be correct? :
```[[paths]]
    reference = "en/products/vpn/download/*.ftl"
    l10n = "{locale}/products/vpn/platform-post-download.ftl"
```
Asking in case, because there is `reference = "en/products/vpn/download.ftl"` for the download page